detail.py

```diff
@@ -0,0 +1,31 @@
+from django_mako_plus import view_function, jscontext
+from django.http import HttpResponseRedirect
+from catalog import models as cmod
+import math
+
+def fix_html(x):
```

Comment:
More comments to explain the purpose of this function would be helpful

Reply:
True! I need to work on that.